server/src/main/kotlin/io/spine/server/query/QueryingClient.kt

```diff
@@ -43,40 +43,49 @@ import java.util.*
  *
  * @param T the type of entity states to query.
  */
-public class QueryingClient<T : EntityState<*>>
-constructor(
+public class QueryingClient<T : EntityState<*>>(
     private val context: BoundedContext,
     private val type: Class<T>,
     actorName: String
 ) {
 
     private val actor = userId { value = actorName }
-    private val factory = ActorRequestFactory.newBuilder()
-        .setActor(actor)
-        .build()
+    private val factory = actorRequestFactory(actor)
 
     /**
      * Obtains a state of an entity by its ID.
      *
      * The value of the ID must be one of the [supported types][io.spine.base.Identifier].
      *
-     * @return the state of the entity or empty `Optional` if the entity with the given ID
-     *         was not found
+     * @return the state of the entity or `null`, if the entity with the given ID was not found.
      * @throws IllegalArgumentException
-     *          if the given ID is not of one of the supported types
+     *          if the given ID is not of one of the supported types.
      */
-    public fun find(id: Any): Optional<T> {
+    public fun findById(id: Any): T? {
         Identifier.checkSupported(id.javaClass)
         val query = buildQuery(id)
         val results = execute(query)
-        return if (results.isEmpty()) {
-            Optional.empty()
+        return if (results.isNotEmpty()) {
+            results.theOnly()
         } else {
-            val value = results.theOnly()
-            Optional.of(value)
+            null
         }
     }
 
+    /**
+     * Obtains a state of an entity by its ID.
+     *
+     * The value of the ID must be one of the [supported types][io.spine.base.Identifier].
+     *
+     * @return the state of the entity or empty `Optional` if the entity with the given ID
+     *         was not found.
+     * @throws IllegalArgumentException
+     *          if the given ID is not of one of the supported types.
+     * @see [id]
```

Comment:
Not sure I understand this reference. 

Maybe, that's a typo, and you wanted to reference an alternative method? If not, and you really wanted to reference the `id` parameter, please have some explanatory text next to the reference itself. 

It's the same issue below, in the next newly deprecated method.

Reply:
The method I introduced initially was called `id()`. But it didn't look well in the tests. For example, in a `ProcessManager` it would be like:
```kotlin
val loaded = select(PmQuizStats::class.java).id(id())
```
So I turned it to be a longer name with the verb.

Reply:
I'm fixing the refs.